README.md

```diff
@@ -17,6 +17,17 @@
 -  `container_tag`: The container tag to scan against (default: `latest`)
 -  `crowdstrike_region`: The CrowdStrike Cloud region to submit for scanning (default: `us-1`)
 
+### Optional input for vulnerability score threshold
+
+-  `crowdstrike_score`: The score threshold used to allow for step success (default: `500`)
+
+NOTE: Scoring is based on the CrowdStrike vulnerability severity tabe scoring shown below.
+```
+critical_score = 2000
+high_score = 500
+medium_score = 100
+low_score = 20
+```
```

Comment:
I wonder if we should make this an actual MD table? I think it could cause confusion between yaml syntax to then have something that looks like code variables.